DOCKERHUB-COMPANION.md

```diff
@@ -0,0 +1,21 @@
+# Kroki Companion Image
```

Comment:
I think we should say "Kroki Companion Server".
In my opinion "Image" is redundant since we are on hub.docker.com